### DISCUSSION TITLE
Creating a Multiline Comment in the GitHub Web Editor

### DISCUSSION THREAD

**Discussion by joshjohanning** [community/community#133557] [Discover: GitHub Best Practices]
## Introduction

I'm embarrassed to say that every time I was editing a GitHub Actions workflow file in GitHub's web editor, and if I needed to make a multiline comment, I either:

1. Added a `#` to each line manually (painful!)
2. Cloned the repo and opened it in VS Code to use keyboard shortcuts to create the multiline comment there
3. Opened the repo with the [github.dev web-based editor](https://docs.github.com/en/codespaces/the-githubdev-web-based-editor) using the `.` shortcut and created the multiline comment there

I knew one way to create a multiline comment in VS Code (and the github.dev web-based editor) was (`Cmd ⌘` + `k`, `Cmd ⌘` + `c`), but I didn't know how to do it in the GitHub web editor, or if it was even possible.

Well, TIL!

## The Shortcut

That shortcut is simply `Cmd ⌘` + `/` on macOS (or `Ctrl` + `/` on Windows). It's that easy! This comments and uncomments the selected lines.

It seems as if `Cmd ⌘` + `/` is a standard keyboard shortcut for commenting/uncommenting in many editors, including VS Code, Atom, and Sublime Text. I'm probably just used to `Cmd ⌘` + `k`, `Cmd ⌘` + `c` from my Visual Studio days.

This works for GitHub Actions workflow files, of course, but I found it also works when editing other code-based files such as JavaScript, Go, Python, C#, Java, Ruby, and I'm sure many others.

Happy editing! 🚀 

![multiline-comment](https://github.com/user-attachments/assets/b856234b-7ddd-4019-94c4-42ddb4cd0121)

**Comment by Javokhdev** [comment]
thanks for useful information

**Reply by joshjohanning** [reply]
Glad you found it helpful!

**Comment by mihailkin** [comment]
Спасибо возможно поможет )

**Comment by chefHarun** [comment]
Thank you for information!
